docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
 ## Debugging OOD Job Failures
 
 If your OOD job cards are disappearing after being allocated or during the job, see our [documentation](ood_layout.md#debugging-ood-job-failures) for instructions on how to retrieve the logs and submit a ticket to Research Computing support.
+
+## Choosing the right Jupyter Application
```

Comment:
Capitalize "Right" in heading titles. Most words are capitalized, only prepositions and articles with three or fewer letters are lowercase.